exchanges/ftx/ftx_test.go

```diff
@@ -226,10 +226,23 @@ func TestGetFuture(t *testing.T) {
 
 func TestGetFutureStats(t *testing.T) {
 	t.Parallel()
-	_, err := f.GetFutureStats(context.Background(), "BTC-PERP")
+	perp, err := f.GetFutureStats(context.Background(), "BTC-PERP")
 	if err != nil {
 		t.Error(err)
 	}
+
+	if perp.Greeks != nil {
+		t.Fatal("greeks should not be contained in perpetual")
+	}
```

Comment:
I'm not sure this is worth the error. If they add it in future, is it worth updating tests for?